docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -0,0 +1,220 @@
+          g.stack +
+          { yaxes: g.yaxes('rps') },
+        )
+      )
+      .addRow(
+        g.row('Disk')
+        .addPanel(
+          g.panel('Disk IO Utilisation') +
+          // Full utilisation would be all disks on each node spending an average of
+          // 1 second per second doing I/O, normalize by metric cardinality for stacked charts.
+          g.queryPanel(|||
+            (
+              instance_device:node_disk_io_time_seconds:rate1m{%(nodeExporterSelector)s}
+            / ignoring (instance, device) group_left
+              count without (instance, device) (instance_device:node_disk_io_time_seconds:rate1m{%(nodeExporterSelector)s})
+            )
+          ||| % $._config, '{{instance}} {{device}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+        .addPanel(
+          g.panel('Disk IO Saturation') +
+          g.queryPanel(|||
+            (
+              instance_device:node_disk_io_time_weighted_seconds:rate1m{%(nodeExporterSelector)s}
+            / ignoring (instance, device) group_left
+              count without (instance, device) (instance_device:node_disk_io_time_weighted_seconds:rate1m{%(nodeExporterSelector)s})
+            )
+          ||| % $._config, '{{instance}} {{device}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+      )
+      .addRow(
+        g.row('Network')
+        .addPanel(
+          g.panel('Net Utilisation (Bytes Receive/Transmit)') +
+          g.queryPanel(
+            [
+              'instance:node_network_receive_bytes:rate1m{%(nodeExporterSelector)s}' % $._config,
+              '-instance:node_network_transmit_bytes:rate1m{%(nodeExporterSelector)s}' % $._config,
+            ],
+            ['{{instance}} Receive', '{{instance}} Transmit'],
+            legendLink,
+          ) +
+          g.stack +
+          { yaxes: g.yaxes('Bps') },
+        )
+        .addPanel(
+          g.panel('Net Saturation (Drops Receive/Transmit)') +
+          g.queryPanel(
+            [
+              'instance:node_network_receive_drop:rate1m{%(nodeExporterSelector)s}' % $._config,
+              '-instance:node_network_transmit_drop:rate1m{%(nodeExporterSelector)s}' % $._config,
+            ],
+            ['{{instance}} Receive', '{{instance}} Transmit'],
+            legendLink,
+          ) +
+          g.stack +
+          { yaxes: g.yaxes('rps') },
+        )
+      )
+      .addRow(
+        g.row('Storage')
+        .addPanel(
+          g.panel('Disk Capacity') +
+          g.queryPanel(|||
+            (
+              sum without (device) (
+                max without (fstype, mountpoint) (
+                  node_filesystem_size_bytes{%(nodeExporterSelector)s, %(fsSelector)s} - node_filesystem_avail_bytes{%(nodeExporterSelector)s, %(fsSelector)s}
+                )
+              ) 
+            / ignoring (instance) group_left
```

Comment:
Instance label should be the same here, this shouldn't need a group_left

Reply:
As before, the right hand size does not have an `instance` label. It is aggregated away to get the total disk size across instances, which is in turn a normalization for the stacked graph.